### ISSUE TITLE
Can we customize 502 bad gateway response

### ISSUE THREAD

**Issue by aliFetvaci61** [gravitee-io/issues#6317]
Can we customize 502 bad gateway response

**Comment by mouligno** [comment]
Yes you can't customize `GATEWAY_INVALID_RESPONSE` from the response template : https://docs.gravitee.io/apim/3.x/apim_publisherguide_response_templates.html

**Comment by aliFetvaci61** [comment]
When I call an api with context-path /lending as /lendingXXX, 502 bad gateway returns. but I want to fill in the response payload. How Can I Customize? @mouligno

**Comment by mouligno** [comment]
What are you trying to do ? 

If you try to call an API with `/echo`  as context-path using `/echoXX`, you should get a 404 error.
